android/app/src/main/java/org/fossasia/openevent/status/BookmarkStatus.java

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.openevent.status;
+
+import android.widget.ImageButton;
+
+import org.fossasia.openevent.listeners.OnBookmarkSelectedListener;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class BookmarkStatus {
+
+    private ImageButton imageButton;
+    private int storedColor;
+    private int sessionId;
+    private OnBookmarkSelectedListener.Status actionCode;
```

Comment:
Move the status out of listener, you may add it in this class instead. Listener should be pure interface